app/public/static/js/libraries/aframe-inspector.js

```diff
@@ -29894,12 +29894,12 @@ object-assign
           {
             key: "getRequests",
             value: function getRequests(self) {
-              var baseUrl = "https://backend-dev.jaydhulia.com/";
+              var baseUrl = "http://localhost:8888/";
               var apiEndpointScene = AFRAME.scenes[0]
                 .getAttribute("id")
                 .replace("-scene", "");
               var baseEndpoint = "api/admin/v1/";
-              var assetsUrl = "https://backend-dev.jaydhulia.com/static/";
+              var assetsUrl = "http://localhost:8888/static/";
```

Comment:
this is because the inspector and backend run "locally" on EC2?

Reply:
It's for running locally since the purpose of this inspector static file is to make it work locally without having to run inspector, so it needs to talk to localhost backend, rather than https backend